lib/update-config.js

```diff
@@ -234,11 +234,22 @@ module.exports = co.wrap(function * (webpackConfig, options) {
     const srcPath = typeof options.aliasDefault === 'string'
       ? options.aliasDefault
       : 'src'
-    webpackConfig.resolve.modules.push(_.cwd(srcPath))
+    webpackConfig.resolve.alias = {
+      src: _.cwd(srcPath),
+      components: _.cwd(srcPath, 'components'),
+      store: _.cwd(srcPath, 'vuex', 'store.js'),
+      css: _.cwd(srcPath, 'css'),
+      views: _.cwd(srcPath, 'views')
+    }
```

Comment:
Wouldn't that technically be considered a breaking change? I mean, I'm fine with that I suppose, but you'd need to release a major version update.